stl/src/xtime0.cpp

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// This must be as small as possible, because its contents are
+// injected into the msvcprt.lib and msvcprtd.lib import libraries.
+// Do not include or define anything else here.
+// In particular, basic_string must not be included here.
+
+#include <algorithm>
+#include <chrono>

```

Comment:
`<algorithm>` and `<chrono>` are large headers - and `<chrono>` drags in `<string>` via `<format>`. (There are only 2 import libs, for release and debug, but 5 possible IDL settings, and the import lib is effectively statically linked to user code, hence the comment warning about this hazard.) Can we reduce `xtime0.cpp` to dragging in only the essential WinAPI machinery, and any simple types/enums it needs to wrap those APIs, much like `filesystem.cpp` does?

Reply:
Removed `#include <algorithm>` and `#include <chrono>` from `xtime0.cpp`.